### PR TITLE
Create abhayypatel.md

### PR DESCRIPTION
**Pull request by abhayypatel**
## Describe your project in 1-3 sentences

This alert system aims to inform users of significant changes in cryptocurrency values. The raspberry pi will serve as a server to track prices and a mail server to send alerts via email, and will also use text-to-speech to announce price changes. This system can be useful for cryptocurrency traders looking to stay informed about market trends and make informed decisions.

## Hack Club Slack handle

Abhay Patel

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: abhayypatel.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+6. Test the system to ensure that it is working properly and sending alerts as expected. This will involve manually triggering alerts by simulating price changes and verifying that the alerts are sent and received correctly.
+
+7. Deploy the system and begin using it to stay informed about significant price changes in the cryptocurrency of choice. This will involve keeping the Raspberry Pi running and monitoring the alerts as they come in.
+
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+
```
Comment (bellesea) [inline abhayypatel.md:38]:
```suggestion
```

File: abhayypatel.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+| Raspberry Pi   | https://www.amazon.com/Raspberry-Model-2019-Quad-Bluetooth/dp/B07TD42S27?tag=georiot-us-default-20&ascsubtag=tomshardware-us-5967095305306638000-20&geniuslink=true&th=1  | $125.67  |
+| Raspberry Pi Case | https://www.amazon.com/Flirc-Raspberry-Pi-Case-Silver/dp/B07WG4DW52/ref=sr_1_6?keywords=raspberry+pi+4+case&qid=1671454629&s=electronics&sr=1-6   | $15.95 |
+| Raspberry Pi SD Card | https://www.amazon.com/SanDisk-256GB-Extreme-microSD-Adapter/dp/B07FCR3316/ref=sr_1_15?keywords=sd+card&qid=1671462466&sr=8-15   | $33.88 |
+| Raspberry Pi Power Supply | https://www.amazon.com/Raspberry-Model-Official-SC0218-Accessory/dp/B07W8XHMJZ/ref=sr_1_3?keywords=raspberry+pi+4+power+supply&qid=1671460848&s=electronics&sprefix=raspberry+pi+4+powe%2Celectronics%2C56&sr=1-3 | $7.99 |
+| Raspberry Pi RBG, Speakers, etc. | https://www.amazon.com/Freenove-Complete-Raspberry-708-Page-Tutorial/dp/B09ZXNL2WH/ref=sr_1_13?crid=1KOPVMV5FO7ML&keywords=raspberry+pi+4+accessories&qid=1671460872&s=electronics&sprefix=raspberry+pi+4+acc%2Celectronics%2C54&sr=1-13  | $59.95 |
+
```
Comment (bellesea) [inline abhayypatel.md:44]:
```suggestion
```

**Review by bellesea — CHANGES_REQUESTED** [review]
@abhayypatel this is really awesome! how do you plan on interacting with / programming the Raspberry Pi?

File: abhayypatel.md (hunk 1 above)
Comment (abhayypatel) [reply abhayypatel.md:38]: > @abhayypatel this is really awesome! how do you plan on interacting with / programming the Raspberry Pi?

Hi there, thanks for the reply! I plan on interacting with the Raspberry Pi by also implementing an alert system which will play a loud alarm sound and flash RBG lights. The Raspberry Pi will also announce the signal through the speaker in TTS as stated earlier. And finally, the Raspberry Pi will manage the backend requests to validate a signal before it announces it. Please let me know if you’d like to know anything else.

**Comment by bellesea** [comment]
@abhayypatel thanks sm for adding the changes! before we move forward, i'd love to know if you have what you need in order to program your Pi... for example, a way to insert/ an SD card on your computer?

**Comment by abhayypatel** [comment]
> @abhayypatel thanks sm for adding the changes! before we move forward, i'd love to know if you have what you need in order to program your Pi... for example, a way to insert an SD card on your computer so you can program it?

Yup, I checked the specifications for what is needed to program the Raspberry Pi and I have the cables at home which are compatible. My apologies for not adding it to the list, I definitely should’ve done that.

**Comment by bellesea** [comment]
> > @abhayypatel thanks sm for adding the changes! before we move forward, i'd love to know if you have what you need in order to program your Pi... for example, a way to insert an SD card on your computer so you can program it?
> 
> Yup, I checked the specifications for what is needed to program the Raspberry Pi and I have the cables at home which are compatible. My apologies for not adding it to the list, I definitely should’ve done that.

No worries at all, just checking to make sure you have all you need hehe!

**Review by bellesea — APPROVED** [review]
@abhayypatel can't wait to see this project in action :) good luck and a member of @hackclub/bank-ops will be reaching out with next steps shortly!

**Comment by abhayypatel** [comment]
> @abhayypatel can't wait to see this project in action :) good luck and a member of @hackclub/bank-ops will be reaching out with next steps shortly!

Sounds good, thanks for your help!

**Review by daisybanks — APPROVED** [review]
@abhayypatel this looks great! The project seems really relevant and modern - super excited to see you build this out during the 10 days of making! Our next step would be to jump on a Zoom call so I can explain how to get you the grant and get you onto Bank. I'm going to move communication over to Slack to make it a bit faster/easier. :)

**Comment by abhayypatel** [comment]
> @abhayypatel this looks great! The project seems really relevant and modern - super excited to see you build this out during the 10 days of making! Our next step would be to jump on a Zoom call so I can explain how to get you the grant and get you onto Bank. I'm going to move communication over to Slack to make it a bit faster/easier. :)

Noted, thanks!

**Comment by abhayypatel** [comment]
ALL ITEMS FINAL RECALCULATED PRICE WITH SHIPPING + TAX:
https://www.amazon.com/gp/product/B09B1JFY24/ref=ox_sc_act_title_1?smid=A14EEANAI4EN1Q&psc=1
https://www.amazon.com/gp/product/B07VFDYNL4/ref=ox_sc_act_title_2?smid=AMIHZKLK542FQ&psc=1
https://www.amazon.com/gp/product/B06W54L7B5/ref=ox_sc_act_title_3?smid=AV3WCPW70PEXC&psc=1
https://www.amazon.com/gp/product/B07TC2BK1X/ref=ox_sc_act_title_4?smid=A2QFBVGOUQ4UGT&psc=1
https://www.amazon.com/gp/product/B07VK9CN98/ref=ox_sc_act_title_5?smid=AHALS71WJO58T&psc=1
FINAL PRICE: $239.72

**Comment by bellesea** [comment]
hey @abhayypatel 👋 could you please open a PR to update your file with the information that you commented above? this would be really helpful for general record keeping, thank you so much!

**Pull request merged**